Corona-Warn-App/src/main/java/de/rki/coronawarnapp/risk/DefaultRiskLevels.kt

```diff
@@ -0,0 +1,249 @@
+        return timeSinceLastDiagnosisKeyFetchFromServer.millisecondsToHours() >
+            TimeVariables.getMaxStaleExposureRiskRange() && isActiveTracingTimeAboveThreshold()
+    }
+
+    override fun calculationNotPossibleBecauseNoKeys() =
+        (TimeVariables.getLastTimeDiagnosisKeysFromServerFetch() == null).also {
+            if (it) {
+                Timber.tag(TAG)
+                    .v("No last time diagnosis keys from server fetch timestamp was found")
+            }
+        }
+
+    override suspend fun calculationNotPossibleBecauseTracingIsOff() =
+        // this applies if tracing is not activated
+        !enfClient.isTracingEnabled.first()
+
+    override suspend fun isIncreasedRisk(): Boolean {
+        val lastExposureSummary = getNewExposureSummary()
+        val appConfiguration = appConfigProvider.getAppConfig()
+        Timber.tag(TAG).v("Retrieved configuration from backend")
+        // custom attenuation parameters to weight the attenuation
+        // values provided by the Google API
+        val attenuationParameters = appConfiguration.attenuationDuration
+        // these are the defined risk classes. They will divide the calculated
+        // risk score into the low and increased risk
+        val riskScoreClassification = appConfiguration.riskScoreClasses
+
+        // calculate the risk score based on the values collected by the Google EN API and
+        // the backend configuration
+        val riskScore = calculateRiskScore(
+            attenuationParameters,
+            lastExposureSummary
+        ).also {
+            Timber.tag(TAG).v("Calculated risk with the given config: $it")
+        }
+
+        // get the high risk score class
+        val highRiskScoreClass =
+            riskScoreClassification.riskClassesList.find { it.label == "HIGH" }
+                ?: throw RiskLevelCalculationException(IllegalStateException("No high risk score class found"))
+
+        // if the calculated risk score is above the defined level threshold we return the high level risk score
+        if (withinDefinedLevelThreshold(
+                riskScore,
+                highRiskScoreClass.min,
+                highRiskScoreClass.max
+            )
+        ) {
+            Timber.tag(TAG)
+                .v("$riskScore is above the defined min value ${highRiskScoreClass.min}")
+            return true
+        } else if (riskScore > highRiskScoreClass.max) {
+            throw RiskLevelCalculationException(
+                IllegalStateException("Risk score is above the max threshold for score class")
+            )
+        }
+
+        return false
+    }
+
+    override fun isActiveTracingTimeAboveThreshold(): Boolean {
+        val durationTracingIsActive = TimeVariables.getTimeActiveTracingDuration()
+        val durationTracingIsActiveThreshold =
+            TimeVariables.getMinActivatedTracingTime().toLong()
+
+        val activeTracingDurationInHours = durationTracingIsActive.millisecondsToHours()
```

Comment:
i would maybe reorder these so that both variables are worked on in order before being compared. This is only personal preference though.

```suggestion
        val durationTracingIsActive = TimeVariables.getTimeActiveTracingDuration()
        val activeTracingDurationInHours = durationTracingIsActive.millisecondsToHours()
        
        val durationTracingIsActiveThreshold =
            TimeVariables.getMinActivatedTracingTime().toLong()
```